docs/_static/css/robots.txt

```diff
@@ -0,0 +1,82 @@
+Sitemap: http://docs.soliditylang.org/sitemap.xml
```

Comment:
Didn't know we have a sitemap, though checking it, it includes all the old versions. Sets a priority to all entries, and the old ones are rather low.

Reply:
Seems to be a RTD feature: https://docs.readthedocs.io/en/stable/hosting.html#sitemaps

Reply:
Looks like it does not support customization though: https://github.com/readthedocs/readthedocs.org/issues/6938#issuecomment-785834511

We would have to roll out our own, probably through a plugin like [sphinx-sitemap](https://github.com/jdillard/sphinx-sitemap).

Reply:
> Sets a priority to all entries, and the old ones are rather low.

Oh, you mean that it will likely not interfere with search engine indexing due to the priorities?